beep/structure/base.py

```diff
@@ -1568,3 +1574,19 @@ def get_CV_current(CV):
     if isinstance(CV, pd.DataFrame):
         if len(CV):
             return(CV.current.iat[-1])
+
+
+def get_CV_capacity(CV):
+    """
+    Helper function to compute CV capacity.
+
+    Args:
+        CV (pd.DataFrame): CV segement of charge
+
+    Returns:
+        (float): charge capacity during the CV segment
+
+    """
+    if isinstance(CV, pd.DataFrame):
```

Comment:
if there is no direct need for this `isinstance`, I'd recommend just removing it. If something gets passed into get_CV_capacity which is not a df, we would want it to throw an error. Currently it would just return `None` and might result in silent, difficult-to-debug, downstream errors